Content.Server/Nyanotrasen/Mail/Components/MailComponent.cs

```diff
@@ -58,43 +55,51 @@ public sealed partial class MailComponent : SharedMailComponent
         /// This can be set to true in the YAML files for a mail delivery to
         /// always be Priority.
         /// </remarks>
-        [DataField("isPriority")]
+        [DataField]
         public bool IsPriority = false;
 
+        // Code imported from New Frontiers, licensed under MIT: large mail
```

Comment:
I see we're already preparing for the Wizden AGPL nuke

Reply:
Literally a thing that Milon told me to word it as when I was originally doing this 2 months ago, and then Frontier became AGPL in the time between that and this PR

Reply:
> I see we're already preparing for the Wizden AGPL nuke

Wizden is NOT going to take any action against AGPL servers is that PJB said. Don't spread misinformation.